components/clp-package-utils/clp_package_utils/scripts/stop_clp.py

```diff
@@ -89,10 +91,19 @@ def main(argv):
 
         if '' == component_name or WORKER_COMPONENT_NAME == component_name:
             stop_container(f'clp-{WORKER_COMPONENT_NAME}-{instance_id}')
+        if '' == component_name or SEARCH_WORKER_COMPONENT_NAME == component_name:
+            stop_container(f'clp-{SEARCH_WORKER_COMPONENT_NAME}-{instance_id}')
         if '' == component_name or SCHEDULER_COMPONENT_NAME == component_name:
             container_name = f'clp-{SCHEDULER_COMPONENT_NAME}-{instance_id}'
             stop_container(container_name)
 
+            container_config_file_path = logs_dir / f'{container_name}.yml'
+            if container_config_file_path.exists():
+                container_config_file_path.unlink()
+        if '' == component_name or SEARCH_SCHEDULER_COMPONENT_NAME == component_name:
+            container_name = f'clp-{SEARCH_SCHEDULER_COMPONENT_NAME}-{instance_id}'
+            stop_container(container_name)
```

Comment:
Should also remove the config file like we do for the normal scheduler.

Reply:
I do already with the container_config_file_path.unlink() just below this if I'm not mistaken?

Reply:
Ah sorry, missed it somehow.